promql/fuzz.go

```diff
@@ -80,13 +92,3 @@ func FuzzParseExpr(in []byte) int {
 
 	return fuzzMeh
 }
-
-// Fuzz the parser.
-func FuzzParseStmts(in []byte) int {
```

Comment:
Could you also remove the reference to this in the comments at the beginning of the file:

    // Repeat for ParseMetricSeletion, ParseExpr and ParseStmt.

Ok, there's more mis-named stuff in there, maybe fix that as well :)

Reply:
done